### DISCUSSION TITLE
ubuntu上运行langchain-chatchat rag问答时报错

### DISCUSSION THREAD

**Discussion by Ederich013** [chatchat-space/Langchain-Chatchat#4595] [Q&A] [answered]
模型为glm4-chat，embedding 为bge-m3 modelengine 为transformers
<img width="1096" alt="image" src="https://github.com/user-attachments/assets/9f23e1db-ce50-4075-9eec-3ebdd4c3659d">
使用自带的sample库时可以回答 使用自建库时报错
多功能对话也有相同报错

**Comment by Ederich013** [comment]
更新一下 linux系统model engine可能不能使用transformer，改为vllm后多功能对话可以使用 但使用自建库时还是报错

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
降低一下transformer版本，最新一般太高了
